TaskVlopper/Models/TaskViewModels.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web;
+using TaskVlopper.Base.Model;
+
+namespace TaskVlopper.Models
+{
+    public class TasksViewModel
```

Comment:
We can refactor Tasks to Task, can't we @bojakowsky? In the matter of fact those are all singular classes so…